app/views/events/_applicants_overview.html.erb

```diff
@@ -1,6 +1,10 @@
 <div class="container-fluid" id="applicants_overview">
   <h3><%= t '.title', title: @event.name %></h3>
-  <div id="free_places"><%= t '.free_places', count: @free_places %></div>
+  <% if @free_places == Float::INFINITY %>
```

Comment:
if you even check for infinity here, why not also setting max_participants to infinity if it is unlimited?!